dev/src/components/Results.tsx

```diff
@@ -47,25 +43,16 @@ const Results = (): JSX.Element => {
         }
     }
 
-    const results = state.courseSubmissions[courseID].map(link => {
-        const data: Row = []
-        if (link.user && link.enrollment) {
-            data.push({ value: link.user.getName(), link: `https://github.com/${link.user.getLogin()}` })
-            data.push(link.enrollment.hasGroup() ? (link.enrollment.getGroup() as Group)?.getName() : "")
-            if (link.submissions) {
-                for (const submissionLink of link.submissions) {
-                    data.push(getSubmissionCell(submissionLink))
-                }
-            }
-        }
-        return data
-    })
+    const base = groupView ? ["Name"] : ["Name", "Group"]
```

Comment:
Shouldn't the `groupView === true` lead to `["Name", "Group"]`? Looks like the opposite is the case now.

Reply:
When `groupView === true`, the `Name` column will contain the group name rather than student name, hence `"Group"`, which displayed the group name, is not needed.

Group view:
![image](https://user-images.githubusercontent.com/1346291/149667144-22214e36-02f9-43e6-b930-3023bb9549fa.png)

Student (individual) view:
![image](https://user-images.githubusercontent.com/1346291/149667110-9e41b312-afa6-441d-8dca-1373d1ce4693.png)